src/admin.js

```diff
@@ -27,6 +27,13 @@ api.bind( 'ready', () => {
 	// Toggle icons when customizer toggles preview mode
 	$( '.collapse-sidebar' ).on( 'click', () => send( 'cdm-toggle-visible' ) );
 
+	// Make the site title clickable
+	$( '.customize-info .site-title' ).on( 'click', () => {
+		if (api.previewer) {
```

Comment:
Small detail, but there should be a space inside these parenthesis. If your editor supports running eslint, this should be marked as a warning (you can also run eslint manually on the file, but it's less convenient).